proxy/cc-proxy.spec-template

```diff
@@ -74,23 +67,21 @@ ln -s %{_builddir}/%{name}-%{version} $HOME/rpmbuild/BUILD/go/src/%{IMPORTNAME}
 cd $HOME/rpmbuild/BUILD/go/src/%{IMPORTNAME}
 make LIBEXECDIR=%{LIBEXECDIR} VERSION=@VERSION_STRING@
 
-%check
-export http_proxy=http://127.0.0.1:9/
-export https_proxy=http://127.0.0.1:9/
-export no_proxy=localhost
+%clean
+echo "Clean build root"
+rm -rf %{buildroot}
 
 %install
 make install DESTDIR=%{buildroot} LIBEXECDIR=%{LIBEXECDIR} VERSION=@VERSION_STRING@
 
+%post
+systemctl stop cc-proxy
```

Comment:
For parity with `debian.postinst`, shouldn't this stop `cc-proxy.socket` and `cc-proxy.service`?

Also, there is the question of whether this post script and the debian one run with `set -e`. I don't believe the debian one does (as it's a stand-alone script and you haven't specified `set -e`). However, for RPM it seems that it depends on the local rpm config.

As we want to ensure the unit files get deleted regardless of whether there is a problem stopping the service, I think we should either add `|| true` after all commands in both the spec file and `debian.rules`. But it might be better (simpler and clearer) if we instead just put this at the top of each script/scriptlet:

```
# XXX: paranoia - ensure that *all* the following commands run.
set +e
```

Reply:
Your're right! `set +e` will do the trick